deselected_tests.yaml

```diff
@@ -437,6 +437,9 @@ gpu:
   # Segfaults
   - ensemble/tests/test_weight_boosting.py
 
+  # sklearn doesn't expect some estimators to support n_jobs
+  - tests/test_common.py::test_check_param_validation[PCA()]
+
```

Comment:
Please take a look on https://github.com/intel/scikit-learn-intelex/pull/1622/files#diff-d1d6209d0bbe078d9a776ecccb3d88fc9de596217d92aaf4b4ae58839f023ac3

Reply:
Same standard was used, let's see the ci at the current state of the PR http://intel-ci.intel.com/eeaf83c4-65b3-f1e4-8adc-a4bf010d0e2e